dali/test/dali_test_resize.h

```diff
@@ -62,7 +62,7 @@ class GenericResizeTest : public DALISingleOpTest<ImgType> {
 
       // determine resize parameters
       if (useExternSizes) {
-        const int *t = ws->Output<CPUBackend>(1)->tensor<int>(i);
+        const auto *t = ws->Output<CPUBackend>(1)->tensor<int>(i);
```

Comment:
Why? auto is useful if you have a complicated type but here it is very simple int.

Reply:
It was the fix for following warning from compiler:
```
Warning:(65, 9) Clang-Tidy: Use auto when initializing with a template cast to avoid duplicating the type name
```